src/Runtime/ExecutionSession.cpp

```diff
@@ -41,6 +41,14 @@ ExecutionSession::ExecutionSession(
     errStr << "Cannot load symbol: '" << entryPointName << "'" << std::endl;
     throw std::runtime_error(errStr.str());
   }
+
+  _inputSignatureFunc = reinterpret_cast<inputSignatureType>(
+      _sharedLibraryHandle.getAddressOfSymbol(inputSignatureName.c_str()));
+  if (!_inputSignatureFunc) {
+    std::stringstream errStr;
+    errStr << "Cannot load symbol: 'omInputSignature'" << std::endl;
```

Comment:
A couple of things:
1) If this is not needed everywhere (e.g. the numerical tests don't need this, but JNI does), don't try to load the function always, but only if the name is set.
2) Since the name CAN be set, it is not always `omInputSignature`, so this error needs to reflect the variable not a hard-coded value.

Reply:
Thanks @sstamenova for the suggestions. @shizy818 can you see if we can make some changes according to the suggestions and make it work for Windows CI? Thanks.

Reply:
@gongsu832 @sstamenova I see. Thanks. Trying...

Reply:
@gongsu832 @AlexandreEichenberger It passes the Windows CI.

Reply:
> Since the name CAN be set, it is not always omInputSignature, so this error needs to reflect the variable not a hard-coded value.

Good point, it is set as a static constant, so it cannot be arbitrarily set during execution, but you are correct, it would be better to reuse the variable in the error message.

Reply:
> If this is not needed everywhere (e.g. the numerical tests don't need this, but JNI does), don't try to load the function always, but only if the name is set.

Is the solution you prefer is to set the pointer to null by default, and then before using it, we first check if its null and when it is, we load it then? Easy to do if that is what you are requesting @sstamenova

Reply:
It depends on what the goal is. If the expectation is that this function (two, in the other PR) will only be used sometimes, then there should be a different mechanism to find them. For example, instead of initializing them immediately, initializing them in a separate function that gets invoked when needed.

I think, though, that this PR is obsolete with your other change, is it not?